eduvpn/network.py

```diff
@@ -11,10 +11,15 @@
 from .server import ConfiguredServer as Server
 from .utils import run_in_background_thread
 
-
 logger = logging.getLogger(__name__)
 
 
+# gettext _(...) function is not available here, but we
```

Comment:
Maybe nicer to make this into a kind of translated property:
```
def translated_property(text):
    return property(lambda self: gettext(text))
```

Reply:
You'd have to tell `xgettext` about this when creating the `.mo` file: `xgettext --keyword=translated_property`.

Reply:
Since `xgettext --keyword=translatable` is already used, I would suggest:
```python
def translatable(text):
    return property(lambda self: gettext(text))
```
Okay?

Reply:
This is getting into bikeshedding territory, but `translatable` sounds to me like the user of this property still need to do something to get the actual translation.